### PR TITLE
chore: update changelog doc

### PR DESCRIPTION
**Pull request by vinckr**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/ecosystem/changelog.md [hunk 1]
```diff
@@ -3,27 +3,32 @@ id: changelog
...
-  [full changelog](https://github.com/ory/kratos/blob/master/CHANGELOG.md)
-- Ory Hydra: [releases](https://github.com/ory/hydra/releases),
-  [full changelog](https://github.com/ory/hydra/blob/master/CHANGELOG.md)
-- Ory Keto: [releases](https://github.com/ory/keto/releases),
-  [full changelog](https://github.com/ory/keto/blob/master/CHANGELOG.md)
-- Ory Oathkeeper: [releases](https://github.com/ory/oathkeeper/releases),
-  [full changelog](https://github.com/ory/oathkeeper/blob/master/CHANGELOG.md)
+## Changelogs
+
+- [Ory Ecosystem Changelog](https://changelog.ory.sh/)
```
Comment (aeneasr) [inline docs/ecosystem/changelog.md:27]:
```suggestion
- [Ory Network Changelog](https://changelog.ory.sh/)
```

File: docs/ecosystem/changelog.md [hunk 2]
```diff
@@ -3,27 +3,32 @@ id: changelog
...
 > announcing important software updates via the [Ory Security Newsletter](http://eepurl.com/di390P). Never miss an update and
 > **[sign up now to important release updates!](http://eepurl.com/di390P)**
 
-:::warning
+:::tip
 
-Changelogs contain important information about the changes and breaking changes introduced in the code. Please read the relevant
-project's changelog before you [upgrade](./upgrading.md) to a new version.
+Looking for support? Please contact us at support@ory.sh!  
+Ory offers support agreements for self-hosted Ory software.
```
Comment (aeneasr) [inline docs/ecosystem/changelog.md:21]: Make this specifically about the getting help with upgrades of selfhosted?

File: docs/ecosystem/changelog.md [hunk 3]
```diff
@@ -3,27 +3,32 @@ id: changelog
 title: Changelog and roadmap
 ---
 
-## Changelog
+## Ory Changelog
+
+:::info
+
+Read the changelog for the Ory ecosystem at [changelog.ory.sh](https://changelog.ory.sh/) !
```
Comment (aeneasr) [inline docs/ecosystem/changelog.md:10]:
```suggestion
Read the changelog for Ory Network at [changelog.ory.sh](https://changelog.ory.sh/) !
```

File: docs/ecosystem/changelog.md (hunk 2 above)
Comment (vinckr) [reply docs/ecosystem/changelog.md:21]: Not sure what you mean, it is specifically for support for self-hosted, no?

Comment (aeneasr) [reply docs/ecosystem/changelog.md:21]: Looks like I can't read

**Pull request merged**